src/roles/vsc-deploy/tasks/main.yml

```diff
@@ -48,11 +48,15 @@
   vars:
     config_template: config.cfg.j2
     reboot_vsc: true
-  when: target_server_type is match("kvm") or target_server_type is match("vcenter") or target_server_type is match("none")
+  when: target_server_type is match("kvm") or target_server_type is match("vcenter") or target_server_type is match("openstack") or target_server_type is match("none")
```

Comment:
Since the when here and on line 58 are the same, can you put them in a block that shares the same when?